code/controllers/configuration/entries/game_options.dm

```diff
@@ -445,3 +445,12 @@
 	default = list("0" = 10, "1" = 10, "2" = 3, "2.5" = 1)
 	key_mode = KEY_MODE_TEXT
 	value_mode = VALUE_MODE_NUM
+
+// Configs for the Quirk system
+/// Disables Quirk point balancing for the server and clients.
+/datum/config_entry/flag/disable_quirk_points
+
+/// The maximum amount of positive quirks one character can have at roundstart.
+/datum/config_entry/number/max_positive_quirks
+	default = -1
```

Comment:
This means that all existing servers are going to have unlimited quirks. You need to default this to 6

Reply:
Fixed by commit [74b561c](https://github.com/tgstation/tgstation/pull/81033/commits/74b561cdc7435eae66da2f589f41ef6c5c3fdcc5)